pyqt-apps/siriushla/widgets/log_label.py

```diff
@@ -24,14 +29,17 @@ class PyDMLogLabel(QListWidget, TextFormatter, PyDMWidget, DisplayFormat):
     errorcolor = QColor(255, 0, 0)
     warncolor = QColor(200, 200, 0)
 
-    def __init__(self, parent=None, init_channel=None):
+    def __init__(self, parent=None, init_channel=None, replace=list()):
```

Comment:
@anacso17 , I try to avoid using default argument values that are mutable. Instead of passing mutable as default arguments I use `None` and set the desired default value inside de implementation, while checking for `None`. for non mutable values, like `int`or `double` I use the value itself, instead of `None`

https://docs.python-guide.org/writing/gotchas/